spec/requests/authentications_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+require 'openid_connect'
+require 'json'
+
+RSpec.describe "Authentications", type: :request do
+  before() do
```

Comment:
[MISC] Unneeded parenthesis

```suggestion
  before do
```